scripts/us_eia/eia_860/2_plant.mcf

```diff
@@ -0,0 +1,232 @@
+# Schema for new properties describing EIA Power Plants
```

Comment:
This is the same file in schema directory, so should we drop it from here (to avoid possible divergence)  ?

Reply:
yes, i was wondering about that as well. it's less ideal for the local import process though; maybe i can try adding the next set through the schema repo.